### ISSUE TITLE
Class not used warning when used as a function param

### ISSUE THREAD

**Issue by CJStroud** [Gert-dev/php-integrator-linter#22]
Hey,

First I love the unused use statement warning, so helpful!

I found an issue where the linter thinks a class isn't being used if it is within a function call.

This line has the warning **_Rhumsaa\Uuid\Uuid is not used anywhere._**

```
$prefix = substr(Uuid::uuid4()->toString(), 0, 9);
```

Doing this stops the warning.
```
$uuid = Uuid::uuid4()->toString();
$prefix = substr($uuid, 0, 9);
```

**Comment by Gert-dev** [comment]
Thanks for reporting this. This problem is actually occurring in the base package and I think I've fixed it there. I'm going to test it further for a bit and if I find no more problems, I will release 0.7.1 which contains a fix for this.

**Comment by Gert-dev** [comment]
Should be fixed in 0.7.2! If you experience any other issues, you're always welcome to open more tickets.
